homeassistant/components/rest_command.py

```diff
@@ -72,6 +74,9 @@ def async_register_rest_command(name, command_config):
             template_payload = command_config[CONF_PAYLOAD]
             template_payload.hass = hass
 
+        content_type = command_config[CONF_CONTENT_TYPE]
+        headers = {HTTP_HEADER_CONTENT_TYPE: content_type}
```

Comment:
I think we should not set a default content type and force overwrite aiohttp internals. So if no type is get we should set headers None